src/components/Plot/Plot.js

```diff
@@ -74,35 +74,20 @@ export const Plot = ({
   x,
   y,
 
-  image = getPlotImage(plotContent),
+  image = getPlotImage(plotContent, x, y),
   lifeStage = plotContent &&
     getPlotContentType(plotContent) === itemType.CROP &&
     getCropLifeStage(plotContent),
-  isRipe = lifeStage === cropLifeStage.GROWN,
+  isRipe = lifeStage === cropLifeStage.GROWN ||
+    (plotContent && getPlotContentType(plotContent) === itemType.WEED),
 }) => {
   const item = plotContent ? itemsMap[plotContent.itemId] : null
   const daysLeftToMature = getDaysLeftToMature(plotContent)
   const isCrop =
     plotContent && getPlotContentType(plotContent) === itemType.CROP
   const isScarecow = itemsMap[plotContent?.itemId]?.type === itemType.SCARECROW
-  const [wasJustShoveled, setWasJustShoveled] = useState(false)
-  const [initialIsShoveledState, setInitialIsShoveledState] = useState(
-    Boolean(plotContent?.isShoveled)
-  )
-
-  useEffect(() => {
-    if (
-      !initialIsShoveledState &&
-      plotContent?.isShoveled &&
-      plotContent?.oreId
-    ) {
-      setWasJustShoveled(true)
-    }
-  }, [initialIsShoveledState, plotContent])
```

Comment:
I think I know why this was here now. I found a newly-introduced bug wherein the "mined" animation replays when you leave the Field and then come back:

[Screencast from 07-20-2022 09:19:21 PM.webm](https://user-images.githubusercontent.com/366330/180116267-e9e7d6dd-721f-4ce1-a2af-5bd0e025ec59.webm)

We'll have to figure out a solution for this before we can merge. :confused: